core/tests/protractor_utils/AdminPage.js

```diff
@@ -34,6 +34,13 @@ var AdminPage = function() {
   var roleSelect = element(by.css('.protractor-update-form-role-select'));
   var statusMessage = element(by.css('[ng-if="$ctrl.statusMessage"]'));
 
+  // Viewing roles can be done by two methods: 1. By roles 2. By username
+  var viewRoleMethod = element(by.css('.protractor-test-view-role-method'));
+  var viewRoleByRoleValue = element(by.css('.protractor-test-view-role-value'));
+  var viewRoleByUsername = element(by.css(
```

Comment:
do not start the name of the selector with a verb if possible as it looks like a function instead.

Also, what's the `viewRoleMethod`?

Reply:
@DubeySandeep i have added the comment for viewRoleMethod (it's a dropdown actually). I'll rename it to something with a dropdown

Reply:
Done